### ISSUE TITLE
Cannot download catalina copy MBP early 2011 13"

### ISSUE THREAD

**Issue by gluehbirne123** [dosdude1/macos-catalina-patcher#108]
Hi,

i try to download a copy of catalin for MBP early 2011 13". I get an error "Could not locate the correct entry in the CatalogURL. Cannot continue." I am on 10.13.6 right now, if that matters.

**Comment by jhallstr** [comment]
> Hi,
> 
> i try to download a copy of catalin for MBP early 2011 13". I get an error "Could not locate the correct entry in the CatalogURL. Cannot continue." I am on 10.13.6 right now, if that matters.

That's the message you'll get when you try download a version of MacOS from Apple which is not supported on the Mac you're trying to download it to.  Yes, Apple uses your Mac's ID to prevent this.  To download a version of MacOS, you need access to a Mac on which that version is supported, and download it fro Apple there.  Then you can transfer it to the Mac you want to use patcher to install it on.
However the exception seems to be using the patcher tool to download the latest version seems to work despite this issue.  But you can't a version earlier than the latest version.

**Comment by gluehbirne123** [comment]
Hi,

i do not understand exactly... of course my macbook is not supported for catalina - that´s why i want to use catalina patcher. in the video there is the same situation - unsupported mac downloads a copy of catalina in one of the first steps. this fails on my mac.

**Comment by jhallstr** [comment]
Well, if you're using the patcher's download option it SHOULD work and give you the latest version.  I guess I was too close to my problem where the latest version created a problem with my wifi, and I was trying to get an earlier version.  Thats where you run into this problem.  But if you want the latest, it should work.  hmm which version of patcher do you have installed?  The Catalina Patcher 1.4.4 is the one to use for the latest Catalina versions.

**Comment by gluehbirne123** [comment]
Yes, i try it with 1.4.4 - i do not know what the problem is. Logs are empty.

**Comment by jhallstr** [comment]
Well, I have no idea then.  Sounds like you need dosdude1's help.  Good luck.

**Comment by jhallstr** [comment]
I just checked, patcher 1.4.4 running on Mojave can still download the latest Catalina.

**Comment by gluehbirne123** [comment]
hopefully dosdude can held me... i just updated to mojave - download of catalins still does not work.
